README.md

```diff
@@ -37,9 +37,10 @@ If you would like to import the govuk elements Sass files into your project, you
 
     `npm install govuk-elements-sass`
 
-This will install the files within `public/sass` to `node_modules/govuk-elements-sass`.
+This will install the files within `public/sass` to `node_modules/govuk-elements-sass` and the govuk_frontend_toolkit to
```

Comment:
This is total reckon but this read a bit odd to me because the fact the package is an NPM package seems implied.

It makes more sense as a concept to me to say something like: "The `govuk-elements-sass` files will be installed into your `node_modules` folder as `node_modules/govuk-elements-sass`. The files included in this package are restricted so only those in the `public/sass` folder will be copied across.

This is just my opinion though and I'm far from the typical user so be interested to hear what others think of this.